src/Umbraco.Web/PropertyEditors/EmailAddressConfiguration.cs

```diff
@@ -7,7 +7,7 @@ namespace Umbraco.Web.PropertyEditors
     /// </summary>
     public class EmailAddressConfiguration
     {
-        [ConfigurationField("IsRequired", "Required?", "boolean")]
+        [ConfigurationField("IsRequired", "Required?", "hidden", Description = "Deprecated; Make this required by selecting mandatory when adding to the document type")]
```

Comment:
```suggestion
        [ConfigurationField("IsRequired", "Required?", "hidden", Description = "Deprecated; Make this required by selecting mandatory when adding to the document type")]
        [Obsolete("No longer used, use `Mandatory` for the property instead. Will be removed in the next major version")]
```

Reply:
Will need to add `using System;` at the top too.

Reply:
Won't marking as `Obsolete` hide it if the `showDeprecatedPropertyEditors` in `umbracosettings.config` is set to true?

Reply:
This is only obsoleting the `IsRequired` property -  https://github.com/umbraco/Umbraco-CMS/pull/8655/files#diff-54cf08e9c3a954b718f21f33ca16968aR11